components/ProjectItem/index.tsx

```diff
@@ -0,0 +1,175 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import Image from 'next/image';
+import {
+  queryDefaultImages,
+  queryProjectbyId,
+} from '@/api/supabase/queries/query';
+import {
```

Comment:
pull from main + rebase then use the icons that Neha has imported. Check her PR for how she added the colors for the renewable energy technology icons: https://github.com/calblueprint/ace-ny/pull/63

You can also use the new components she made for renewable energy technology and status (if it works for your sprint -- I am not sure if the fonts / text are the same)